kastle/connection.py

```diff
@@ -0,0 +1,88 @@
+import asyncio
+from typing import Dict
+
+from . import errors
+
+CRLN = b"\r\n"
```

Comment:
```suggestion
CRLF = b"\r\n"
```
The correct term is *Carriage Return and Line Feed*.